app/controllers/api/v1/cocktails_controller.rb

```diff
@@ -26,6 +26,81 @@ def index
       end
   end
 
+  def show
+    response = RestClient.get(
+      "https://www.thecocktaildb.com/api/json/v1/1/lookup.php",
+      {
+        params: {
+          i: params[:id]
+        }
+      }
+    )
+    drinks = JSON.parse(response)
+    sanitized_drinks = drinks["drinks"].map { |drink|
+      {
+        "id": drink["idDrink"],
+        "name": drink["strDrink"],
+        "category": drink["strCategory"],
+        "IBA": drink["strIBA"],
+        "glass": drink["strGlass"],
+        "instructions": drink["strInstructions"],
+        "image": drink["strDrinkThumb"],
+        ingredients: [
+          { name: drink["strIngredient1"],
+           measure: drink["strMeasure1"],
+         },
```

Comment:
The formatting seems a little off here with the closing curly bracket but it might be fine if that's what the code formatter output. But the brackets further down look fine, its just the first two that look off

Reply:
We formatted this code now. Updated the PR

Reply:
The formatting locally doesn't look like here in Github. We pushed already a couple of times adding random comments but it looks the same.